zmon-agent.py

```diff
@@ -158,8 +157,8 @@ def get_running_apps(region):
                 ins['runtime'] = user_data['runtime']
 
                 if 'StackVersion' in tags:
-                    ins['stack'] = tags['Name']
-                    ins['resource_id'] = tags['aws:cloudformation:logical-id']
+                    ins['stack'] = tags.get('Name', None)
+                    ins['resource_id'] = tags.get('aws:cloudformation:logical-id', None)
```

Comment:
I think all entity fields must be strings, i.e. having "None" would not work in ZMON AFAIK (or might cause problems)